### ISSUE TITLE
TOR support?

### ISSUE THREAD

**Issue by MrSimmo** [SchizoDuckie/DuckieTV#1128]
With the level of torrent blocking ever increasing, would it be possible to see if there is a way to search for magnets/torrent sources using TOR? 

A manual method people currently use is:
1. Use TOR browser to browse to the appropriate website/onion version of the website
2. Search then Copy/Paste magnet link into Deluge

**Comment by 2635599** [comment]
what is wrong with adding a vpn like nord to your system or browsec to your browser and only use the browser extensions for dtv?

**Comment by MrSimmo** [comment]
Theres nothing wrong with either but this would give an in-the-box solution.

**Comment by 2635599** [comment]
fair enought as long if implemented it can be toggled on and off.

**Comment by zero77** [comment]
@MrSimmo 
I dont think using tor will help because lots of the tor exit IPs are blocked by DDoS mitigation services like Cloudflare and will require you to fill in a capture.

you are better off using Jackett with a proxy enabled, because Jackett gives you the option to test each indexer/website to see if its working with or without a proxy.

https://github.com/Jackett/Jackett

**Comment by MrSimmo** [comment]
Agreed, it could work with the .onion version of the sites though which would save exits? Agreed though the whole Captcha thing is pita..

**Comment by zero77** [comment]
> With the level of torrent blocking ever increasing

I am finding the same thing although, i think Jackett with https://github.com/Jackett/Jackett/issues/2359 and https://github.com/Jackett/Jackett/issues/2539 implemented will be a better solution than tor.

> Agreed, it could work with the .onion version of the sites though which would save exits?

For the sites that have .onion versions this may work but, i haven't tried it.

**Comment by MrSimmo** [comment]
👍

**Comment by ghost** [comment]
afaik, accessing through `.onion` doesn't prompt captchas at least for cloudflare, because cloudflare has "onion services" enabled by default
